CHANGELOG.md

```diff
@@ -1,4 +1,124 @@
-# Change Log
+# Changelog
+
+## 2016-10-25 Release [0.5.0](https://github.com/voxpupuli/puppet-nginx/tree/0.5.0)
```

Comment:
we're parsing this line with a regex. I'm not sure if it matches when the number is surrounded by braces.